src/client/src/components/pages/loading/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./index.css";
+
+export default function index() {
```

Comment:
If you switch this function name to something more descriptive, like `loader` or something, you'd see that name inside the [react devtools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en) extension for chrome, in its component inspector, which can be pretty useful.